.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,13 @@
+Proposed commit message:
+
+```
+${action} (${issues} / ${pull-request}) [max 70 characters]
```

Comment:
Did you don't write the `#` by purpose (like `#357`) or is it related to the contributing?

Reply:
I basically just copied the template from here:

https://github.com/junit-pioneer/junit-pioneer/blob/master/CONTRIBUTING.md#commit-message

What do you have in mind exactly? Precede e.g. `${issues}` with `#`?

Reply:
Yes that is what I was thinking of. But if we have this in the `CONTRIBUTING` this way I don't know if I just feel to bad about people.